src/OpenRiaServices.Client/Test/Client.Test/Data/EntityContainerTests.cs

```diff
@@ -4465,7 +4465,7 @@ public void AddReferenceWithIncorrectType()
     public class ChangeTrackingTestContext : DomainContext
     {
         public ChangeTrackingTestContext()
-            : base(new WebDomainClient<TestDomainServices.LTS.Catalog.ICatalogContract>(TestURIs.LTS_Catalog))
+            : base(DomainClientFactory.CreateDomainClient(typeof(TestDomainServices.LTS.Catalog.ICatalogContract), TestURIs.LTS_Catalog, false))
```

Comment:
@erikoijwall  @SandstromErik This allows the test to run with the DomainClientFactory configured in "Main.cs" instead of beeing hardcoded to a specific type. (WebDomainClient is also not supported for net6).

In general 
For tests using WebDomainClient there are 2 options when moving forward:
- 1: If they test an implementation detail that is not relevant for other DomainCelients they can continue to be hardcoded and be made conditional for framework only "#if FRAMEWORK"
- 2: They they test som general functionality do as above so the DomainClientFactory  setup in Main is tested